packages/iov-keycontrol/types/connection.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Stream } from "xstream";
```

Comment:
I think it is time to stop checking-in those types. Never used then for anything in weeks. However, this is independent of this PR

Reply:
Yes, when you are back, let's take a few days to revisit the build processes, etc...
Not sure why the special `types` dir is needed outside of the `build` dir. I know there was an issue earlier, but not sure what.